envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/listeners/filters/EnvoyDefaultFilters.kt

```diff
@@ -18,7 +20,8 @@ class EnvoyDefaultFilters(private val snapshotProperties: SnapshotProperties) {
     val envoyRouterHttpFilter = envoyRouterHttpFilter()
     val defaultEnvoyRouterHttpFilter = { _: Group -> envoyRouterHttpFilter }
     val defaultEgressFilters = listOf(defaultHeaderToMetadataFilter, defaultEnvoyRouterHttpFilter)
-    val defaultIngressFilters = listOf(defaultEnvoyRouterHttpFilter)
+    val defaultRbacFilter = { group: Group -> rbacFilterFactory.createHttpFilter(group) }
```

Comment:
Default filters is a set of filters applied to a control plane that is build upon EC. This set includes: router filter, egress filter, rbac filter etc. This code is just temporary it will be removed once we create a truly dynamic environment where these filters can be defined in real time via UI or some rest endpoint. They are not private if someone building upon EC would like to customize them, if you really want to I can close them down.